cmd/helm/release_testing.go

```diff
@@ -35,63 +35,44 @@ The argument this command takes is the name of a deployed release.
 The tests to be run are defined in the chart that was installed.
 `
 
-type releaseTestOptions struct {
-	name    string
-	client  helm.Interface
-	timeout int64
-	cleanup bool
-}
-
-func newReleaseTestCmd(c helm.Interface, out io.Writer) *cobra.Command {
-	o := &releaseTestOptions{client: c}
+func newReleaseTestCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
+	client := action.NewReleaseTesting(cfg)
```

Comment:
Look like there is a `action.NewReleaseTesting` and `action.NewTest` which are duplicates

Reply:
Ooh, good catch. Will consolidate.